docs/guide/known-limitations.md

```diff
@@ -28,7 +28,6 @@ you can't compare the arguments in a formula like this:
 * There is no relative referencing in named ranges.
 * The library doesn't offer (at least not yet) the following features:
   * 3D references
-  * Complex numbers
```

Comment:
We can remove "  * Currency data type" from the list as well? https://github.com/handsontable/hyperformula/pull/590#issuecomment-770292976

Reply:
sure